.github/workflows/daily.yml

```diff
@@ -716,9 +720,17 @@ jobs:
       with:
         repository: ${{ env.GITHUB_REPOSITORY }}
         ref: ${{ env.GITHUB_HEAD_REF }}
-    - name: make
+    - name: fix centos7 EOL
       run: |
+        sed -i 's/^mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+        sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
         yum -y install centos-release-scl epel-release
+        yum clean all
+        rm -rf /var/cache/yum/*
```

Comment:
just to clean the `http://mirror.centos.org` baseurl in the yum cache generated by `centos-release-scl`.

Reply:
why isn't that necessary in the above `test-centos7-jemalloc`, and in ci.yml?
also, why isn't it done between the `sed` lines, and the above `yum install`?

Reply:
in fact, the two answers are the same, after `yum install centos-release-scl`, it will generate the yum cache, we need to clean the cache after it to avoid using the baseurl in the cache, and update the baseurl in CentOS-SCLo*.repo.
test-centos7-jemalloc and in ci.yml doesn't install centos-release-scl.

Reply:
the fix steps:
1. fix `/etc/yum.repos.d/CentOS-*` to install `centos-release-scl epel-release`.
2. clean the cache generated by `yum -y install centos-release-scl epel-release`.
3. fix `/etc/yum.repos.d/CentOS-SCLo*.repo` to install `devtoolset-7`.